src/actinia_core/resources/ephemeral_processing_with_export.py

```diff
@@ -241,14 +239,29 @@ def _export_raster(self, raster_name,
             self._update_num_of_steps(1)
             self._run_module(p)
 
+        if format == 'COG':
+            # check if GDAL has COG driver
+            from osgeo import gdal
+            driver_list = [gdal.GetDriver(i).ShortName for i in range(gdal.GetDriverCount())]
+            if 'COG' not in driver_list:
+                format = 'GTiff'
+                self.message_logger.info("COG driver not available, using GTiff driver")
+
         # Save the file in the temporary directory of the temporary gisdb
         output_path = os.path.join(self.temp_file_path, file_name)
 
-        # generate overviews with compression:
-        os.environ['COMPRESS_OVERVIEW'] = "LZW"
         module_name = "r.out.gdal"
-        args = ["-fmt", "input=%s"%raster_name, "format=%s"%format,
-                "createopt=COMPRESS=LZW,TILED=YES", "overviews=5", "output=%s"%output_path]
+        args = ["-fmt", "input=%s" % raster_name, "format=%s" % format, "output=%s" % output_path]
+
+        if format == "GTiff":
+            # generate overviews with compression:
+            os.environ['COMPRESS_OVERVIEW'] = "LZW"
+            args.extend(["createopt=COMPRESS=LZW,TILED=YES", "overviews=5"])
+
+        # current workaround due to color table export
+        # TODO: DELETE AND TEST ONCE GDAL 3.2 HAS BEEN RELEASED
```

Comment:
```suggestion
        # TODO: DELETE AND TEST ONCE GDAL 3.1.4 HAS BEEN RELEASED
```
See: https://github.com/OSGeo/gdal/issues/2946